.yamllint

```diff
@@ -0,0 +1,58 @@
+---
+
+yaml-files:
+  - '*.yaml'
+  - '.yamllint'
+
+rules:
+  anchors:
+    forbid-undeclared-aliases: true
+    forbid-duplicated-anchors: true
+    forbid-unused-anchors: true
+  braces:
+    forbid: false
+    min-spaces-inside: -1
+    max-spaces-inside: -1
+    min-spaces-inside-empty: -1
+    max-spaces-inside-empty: -1
+  brackets:
+    forbid: false
+    min-spaces-inside: -1
+    max-spaces-inside: -1
+    min-spaces-inside-empty: -1
+    max-spaces-inside-empty: -1
+  colons:
+    max-spaces-before: 0
+    max-spaces-after: -1
+  commas: enable
+  comments:
+    require-starting-space: true
+    ignore-shebangs: true
+    min-spaces-from-content: 1
+  comments-indentation: enable
+  document-end: disable
+  document-start: disable
+  empty-lines:
+    max: 1
+    max-start: 0
+    max-end: 0
+  empty-values: disable
+  float-values: enable
```

Comment:
I think we can disable float values. Bob does not use them. Or do you use some plugin that uses float values?